RELEASES.md

```diff
@@ -1,3 +1,152 @@
+Version 1.45.0 (2020-07-16)
+==========================
+
+Language
+--------
+- [Float to int conversion using `as` has been defined as a saturating conversion.][71269]
+  This was previously undefined behaviour.
```

Comment:
Mind calling out to_int_unchecked here like we did in 1.44?

Reply:
The wording makes it look like any `float as int` operation was a UB, but it was UB only for out-of-range cases.